Assets/Scripts/Game/StaticNPC.cs

```diff
@@ -159,10 +159,19 @@ public void SetLayoutData(DFBlock.RmbBlockPeopleRecord obj, int buildingKey = 0)
         }
 
         /// <summary>
-        /// Sets NPC data from RMB layout flat record. (exterior NPCs)
+        /// Sets NPC data from RMB layout flat record (exterior NPCs).
         /// </summary>
         public void SetLayoutData(DFBlock.RmbBlockFlatObjectRecord obj)
         {
+            // Gender flag is invalid in game data for exterior NPCs: get it from FLATS.CFG instead
+            int flatID = FlatsFile.GetFlatID(obj.TextureArchive, obj.TextureRecord);
+            FlatsFile.FlatData flatCFG;
+            if (DaggerfallUnity.Instance.ContentReader.FlatsFileReader.GetFlatData(flatID, out flatCFG) &&
+                flatCFG.gender.Contains("2"))
+            {
+                obj.Flags |= 32;
```

Comment:
Should we clear the flag if FLATS.CFG says "male", or is it always male by default?

Reply:
It seems to be always male but you're right, it's better to clear the flag, I'll update the PR.